From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Thu, 8 Oct 2026 06:00:36 +0200
Subject: [PATCH] HomePanel: open WebEx and Teams in Atlas

LuneOS does not ship com.webos.app.enactbrowser; its browser is
org.webosports.app.atlas, which takes the same {target: url} launch
parameter. Without this both buttons launch an app that is not
installed and nothing happens.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific: the browser is Atlas]
---
 src/views/HomePanel/HomePanel.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/views/HomePanel/HomePanel.js b/src/views/HomePanel/HomePanel.js
index 32a2d8f..15eaa36 100644
--- a/src/views/HomePanel/HomePanel.js
+++ b/src/views/HomePanel/HomePanel.js
@@ -47,7 +47,7 @@ class HomePanel extends React.Component {
         service: "luna://com.webos.applicationManager",
         method: "launch",
         parameters: {
-          id: "com.webos.app.enactbrowser",
+          id: "org.webosports.app.atlas",
           params: {
             target: "https://signin.webex.com/join/",
             displayAffinity
@@ -68,7 +68,7 @@ class HomePanel extends React.Component {
         service: "luna://com.webos.applicationManager",
         method: "launch",
         parameters: {
-          id: "com.webos.app.enactbrowser",
+          id: "org.webosports.app.atlas",
           params: {
             target: "https://teams.microsoft.com/",
             displayAffinity
